evm/get-duplicate-token-managers.js

```diff
@@ -0,0 +1,120 @@
+const { ethers } = require('hardhat');
+const { Contract, getDefaultProvider } = ethers;
+const info = require('../axelar-chains-config/info/mainnet.json');
+const tokenManagerInfo = require('../axelar-chains-config/info/tokenManagers.json');
+const ITokenManager = require('@axelar-network/interchain-token-service/artifacts/contracts/interfaces/ITokenManager.sol/ITokenManager.json');
+const IInterchainTokenService = require('@axelar-network/interchain-token-service/artifacts/contracts/interfaces/IInterchainTokenService.sol/IInterchainTokenService.json');
+const fs = require('fs');
+const toml = require('toml');
+
+const RPCs = toml.parse(fs.readFileSync('./axelar-chains-config/info/rpcs.toml', 'utf-8'));
```

Comment:
overriding rpcs is occasionally needed. support this natively for any script. maybe something like if a base evm option `--rpc` is passed to any script, it will read from the `info/${env}-rpc.json` config instead?